.github/workflows/tests.yml

```diff
@@ -62,7 +62,19 @@ jobs:
         uses: actions/checkout@v3
         with:
           persist-credentials: false
-
+      - uses: dorny/paths-filter@v2
+        id: working-directory
+        with:
+          filters: |
+            sdk:
+            - 'covalent/**'
+            dispatcher:
+            - 'covalent_dispatcher/**'
+            functional_tests:
```

Comment:
functional tests should always be run, this filter is not needed

Reply:
I have deleted the filter for `functional_tests`